compiler/qsc_passes/src/replace_qubit_allocation/tests.rs

```diff
@@ -39,17 +39,17 @@ fn test_single_qubit() {
                         functors: empty set
                         body: SpecDecl 3 [22-96]: Impl:
                             Block 4 [45-96] [Type Unit]:
-                                Stmt 17 [59-60]: Local (Immutable):
-                                    Pat 18 [59-60] [Type Qubit]: Bind: Ident 7 [59-60] "q"
-                                    Expr 15 [59-60] [Type Qubit]: Call:
-                                        Expr 14 [59-60] [Type (Unit => Qubit)]: Var: Item 4 (Package 0)
-                                        Expr 16 [59-60] [Type Unit]: Unit
+                                Stmt 17 [55-71]: Local (Immutable):
+                                    Pat 18 [55-71] [Type Qubit]: Bind: Ident 7 [55-71] "q"
+                                    Expr 15 [55-71] [Type Qubit]: Call:
+                                        Expr 14 [55-71] [Type (Unit => Qubit)]: Var: Item 4 (Package 0)
+                                        Expr 16 [55-71] [Type Unit]: Unit
                                 Stmt 9 [80-90]: Local (Immutable):
                                     Pat 10 [84-85] [Type Int]: Bind: Ident 11 [84-85] "x"
                                     Expr 12 [88-89] [Type Int]: Lit: Int(3)
-                                Stmt 20 [59-60]: Semi: Expr 21 [59-60] [Type Unit]: Call:
-                                    Expr 19 [59-60] [Type (Qubit => Unit)]: Var: Item 5 (Package 0)
-                                    Expr 22 [59-60] [Type Qubit]: Var: Local 7
+                                Stmt 20 [0-0]: Semi: Expr 21 [0-0] [Type Unit]: Call:
```

Comment:
Is it expected that this went to `0-0`?

Reply:
Just checking - qubit deallocation _can_ raise an error, will the call stack be formatted properly when the statement / expr spans are `0-0`?

Reply:
The only spans considered in stack traces are for items. The span above is for the hidden qubit deallocation which isn't user code.